tools/qontract_cli.py

```diff
@@ -1070,8 +1082,29 @@ def cidr_blocks(ctx) -> None:
 
     cidrs.sort(key=lambda item: ipaddress.ip_network(item["cidr"]))
 
-    ctx.obj["options"]["sort"] = False
-    print_output(ctx.obj["options"], cidrs, columns)
+    if for_cluster:
+        latest_cluster_cidr = next(
+            (item for item in reversed(cidrs) if item["type"] == "cluster"),
+            None,
+        )
+
+        if not latest_cluster_cidr:
+            print("ERROR: Unable to find any existing cluster CIDR block.")
+            sys.exit(1)
+
+        avail_addr = ipaddress.ip_address(latest_cluster_cidr["to"]) + 1
+
+        print(f"INFO: Latest available network address: {str(avail_addr)}")
+        try:
+            result_cidr_block = str(ipaddress.ip_network((avail_addr, mask)))
```

Comment:
I think this should check if there is any overlap with AWS account VPCs as well. Since we use peerings we must not have overlaps with those as well